lib/cylc/task_pool.py

```diff
@@ -643,6 +643,13 @@ def no_active_tasks(self):
                 return False
         return True
 
+    def unkillable_only(self):
```

Comment:
What about `def has_unkillable_tasks`, which should make it instantly recognisable as a method returning a boolean?

Should also add doc string for a new method.